### PR TITLE
Add force flag to ignore cache

### PR DESCRIPTION
**Pull request by danstewart**
As mentioned in #5

### CODE REVIEW

File: snip.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# Flag to ignore cache
+force=0
```
Comment (whoan) [inline snip.sh]: This variable will remain set as this script is meant to be `source`d.

**Review by whoan — CHANGES_REQUESTED** [review]
Good catch @danstewart. Thanks for your PR.

Let me know if you can work on the requested change; Otherwise, I can do it.
My idea is to make that variable local to some function(s) so it does not affect the _global namespace_. Maybe we can provide the force flag as a parameter (with vaue 0 or 1) to the `__snip__replace_snips` function?

**Comment by danstewart** [comment]
Hey @whoan, how's that?

**Comment by whoan** [comment]
Works great @danstewart . Thanks!

**Pull request merged**